docs/src/_includes/layouts/doc.html

```diff
@@ -90,6 +91,22 @@ <h1>{{ title }}</h1>
                     frozen: rule_meta.docs.frozen,
                     hasSuggestions: rule_meta.hasSuggestions
                     }) }}
+
+                    {% if rule_meta.deprecated %}
+                        {% set deprecated_description %}
+                            <p>This rule was <b>deprecated</b> in ESLint v{{ rule_meta.deprecated.deprecatedSince }}.
+                            
+                            {% if rule_meta.deprecated.replacedBy.length === 0 %}
+                                There is no replacement rule.
+                            {% elif rule_meta.deprecated.replacedBy[0].plugin %}
+                                Please use the <a href="{{ rule_meta.deprecated.replacedBy[0].rule.url }}">corresponding rule</a> in <a href="{{ rule_meta.deprecated.replacedBy[0].plugin.url }}">{{ rule_meta.deprecated.replacedBy[0].plugin.name }}</a>.
+                            {% else %}
+                                Please replace the rule with <a href="{{ rule.meta.deprecated.replacedBy[0].rule.url}}">{{ rule.meta.deprecated.replacedBy[0].rule.name }}</a>
```

Comment:
This doesn't seem to work:

![image](https://github.com/user-attachments/assets/8102c40f-2826-4a8d-9ec8-9ae84e1d6497)

While at this, since the docs sites are versioned, I think the link should be relative (`href` could be just the replacement rule's name).